RELEASE_NOTES.rst

```diff
@@ -1,3 +1,133 @@
+         /MMMMMMMMMMMMMMMMMMMs.       oMMMhyMMMMMMMMMMMMMMMMMMMMMMMMMh`
+          yNMMMMMMMMMMMMMMMMM/        yMMmmMMMMMMMMMmyo///oyNMMMMMMMMMs
+           +mMMMMMMMMMMMMMMMMy         /dMMMMMMMMMd:  ./+/-  :dMMMMMMMd
+           `oMMMMMMMMMMMMMMMMd`        .MMMMMMMMMh  .dMMMMMm:  hMMMMMMM:
+          oNMMMMMMMMMMMMMMMMMM:        :MMMMMMMMM:  hMMMMMMMm  :MMMMMMM/
+        :NMMMMMMMMMMMMMMMMMMMM/        -MMMMMMMMMo  -NMMMMMN/  oMMMMMMM/
+        -mMMMMMMdhdMMMMMMMMMMM/`       `dMMMMMMMMM+   :oso/`  oMMMMMMMM:
+         :NMMMN.   :dMMMMMMMMM+         :MMMMMMMMMMmo:`   .:smMMMMMMMNo
+          .yNMMh:   .mMMMMMMMMd          hMMMMMMMMMMMMMMMMMMMMMMMMMMd.
+             ``       .:////++:           oNMMMMMMMMMMMMMMMMMMMMMMd+`
+                                            -+syhhdmmmNNNNNmdhss/.
+
+This release was brought to you by 65 contributors contributing 141
+commits
+
+The name of this release is BrewBear to highlight the improvements made to
+out dependency management system, including the addition of ``brew`` support
+for mac users. Mac has been an officially supported platform for a long time
+and we would like to draw attention to this often overlooked fact.
+Support has also been added for ``Conda`` and ``cargo`` allowing bears which
+use linters available from those package managers to manage their dependencies.
+
+Initial next generation core code has been merged and will be integrated over
+the coming releases. Until then, please use the current API.
+
+We have added a release candidate system to our release process to allow users
+to easily test changes in the latest pre-release version of coala.
+Instructions for obtaining the rc package will be posted for each minor/major
+release to the coala gitter channel, blog, and twitter. We encourage all users
+interested in checking out new features or helping to test these pre-releases
+on their projects. If you do find a regression in a release candidate, please
+report it to the issue tracker with the word ``Regression`` somewhere in the
+title.
+
+**Breaking Changes**
+
+- coala exit codes have changed. If you relied on coala in scripts or
+  other automated systems please check if it it still works with the new
+  behavior. You can find the new exit codes
+  `here <http://docs.coala.io/en/latest/Users/Exit_Codes.html>`__.
+  `PR #3905 <https://github.com/coala/coala/pull/3905>`_
+
+**General**
+
+- Section Inheritance had been added to the coafile parsing. With this
+  change we are deprecating implicit ``default`` section inheritance.
+  Instead, you can now have definite section inheritance by using the
+  ``.`` in the section title. Append operators are now supported allowing
+  addition of values to inherited section. See the ``Mockup`` section of
+  `cEP-0005 <https://github.com/coala/cEPs/blob/master/cEP-0005.md>`_
+  for a syntax example and `PR #3178 <https://github.com/coala/coala/pull/3178/>`_
+  for more information.
```

Comment:
We should at least have a small documentation section with anchor merged, so we can link to it here and expand it over time.